src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/UserService.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.idam;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.idam.client.IdamClient;
+
+import java.util.Map;
+
+@Service
+public class UserService {
+
+    private final IdamClient idamClient;
+    private final Map<String, Credential> users;
+
+    @Autowired
+    public UserService(IdamClient idamClient, @Value("idam.users") Map<String, Credential> users) {
+        this.idamClient = idamClient;
+        this.users = users;
+    }
+
+    public String getBearerTokenForJurisdiction(String jurisdiction) {
+        Credential user = users.get(jurisdiction);
```

Comment:
what if there is no user in the map for given jurisdiction?

Reply:
will throw an exception

Reply:
```
    /**
     * Returns the value to which the specified key is mapped,
     * or {@code null} if this map contains no mapping for the key.
     *
```

Reply:
i.e. i will change the code

Reply:
and it would get a null pointer exception on the next line too 👅